Formula/zsh-history-enquirer.rb

```diff
@@ -0,0 +1,45 @@
+require "language/node"
+
+class ZshHistoryEnquirer < Formula
+  desc "Zsh plugin that enhances history search interaction"
+  homepage "https://zsh-history-enquirer.zthxxx.me"
+  url "https://registry.npmjs.org/zsh-history-enquirer/-/zsh-history-enquirer-1.3.0.tgz"
+  sha256 "d2b673fdb0621bbbbae1529f42ff0c3184fa8e4f4befbc6bd9fa4b2d95ca4b5f"
+  license "MIT"
+
+  depends_on "node"
+  uses_from_macos "expect" => :test
+
+  on_linux do
+    depends_on "expect" => :test
```

Comment:
```suggestion

```

Reply:
This is covered by `uses_from_macos` above.

Reply:
You will need to move this into the `z` directory and also change your commit message to `zsh-history-enquirer 1.3.0 (new formula)`.